certified-connectors/Upland Panviva/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4932 @@
+{
+  "swagger": "2.0",

```

Comment:
Kindly ensure to follow the below instructions and format the json file. 

https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-4d-json-file-formatting-and-limitations